specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -11029,7 +11033,7 @@
         "dependencyAction": {
           "type": "string",
           "title": "An action that the Batch service performs on tasks that depend on this task.",
-          "description": "The default is 'satisfy' for exit code 0, and 'block' for all other exit conditions. If the job's usesTaskDependencies property is set to false, then specifying the dependencyAction property returns an error. The add task request fails with an invalid property value error; if you are calling the REST API directly, the HTTP status code is 400  (Bad Request).",
+          "description": "Values are: satisfy - Satisfy the tasks dependencies. block - Block the tasks dependencies. The default is 'satisfy' for exit code 0, and 'block' for all other exit conditions. If the job's usesTaskDependencies property is set to false, then specifying the dependencyAction property returns an error. The add task request fails with an invalid property value error; if you are calling the REST API directly, the HTTP status code is 400  (Bad Request).",
```

Comment:
>S [](start = 48, length = 1)

casing is inconsistent - for job action you did not capitalise enum value descriptions